<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1598</width>
    <height>1198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="cityButton1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>410</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Berlin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cityButton3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>1050</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Munchen</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Dresden">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>730</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dresden</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cityButton2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bremen</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cityButton4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>930</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Saarbr√ºcken</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cityButton5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>1030</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stuttgart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="distanceButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>100</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Entfernung</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>130</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allLocationsButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>170</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Alle Orte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myLocationButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>210</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Mein Standort</string>
    </property>
   </widget>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>971</width>
      <height>1246</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>971</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>971</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/navigation_scrollbar/866-8664209_leer-deutschland-karte.png</pixmap>
    </property>
   </widget>
   <widget class="QRadioButton" name="meinOrt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>880</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>h-da</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1598</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Laden</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
